packages/adp-tooling/test/unit/preview/adp-preview.test.ts

```diff
@@ -476,13 +476,15 @@ describe('AdaptationProject', () => {
                 middlewareUtil,
                 logger
             );
-            jest.spyOn(helper, 'getVariant').mockReturnValue({
-                content: [],
-                id: 'adp/project',
-                layer: 'VENDOR',
-                namespace: 'test',
-                reference: 'adp/project'
-            });
+            jest.spyOn(helper, 'getVariant').mockReturnValue(
+                Promise.resolve({
+                    content: [],
+                    id: 'adp/project',
+                    layer: 'VENDOR',
+                    namespace: 'test',
+                    reference: 'adp/project'
+                })
+            );
```

Comment:
`jest` also has `mockResolvedValue`, would this work:

```suggestion
            jest.spyOn(helper, 'getVariant').mockResolvedValue(
                {
                    content: [],
                    id: 'adp/project',
                    layer: 'VENDOR',
                    namespace: 'test',
                    reference: 'adp/project'
                }
            );
```

Reply:
adjusted in https://github.com/SAP/open-ux-tools/pull/2931/commits/defc2ba953150d49a6f324a90ec5b7b21a2184c7